package.json

```diff
@@ -97,7 +97,8 @@
     "test": "jest",
     "test:dist": "node ./scripts/test-dist.js",
     "test-integration": "jest tests_integration",
-    "lint": "cross-env EFF_NO_LINK_RULES=true eslint . --format node_modules/eslint-friendly-formatter",
```

Comment:
I think we don't usually format this file. (related: #3671). Should we add `package.json` to `.prettierignore`?

Reply:
I usually do so in my own projects. But I think maybe we should make prettier ignore package.json out of the box.

Reply:
and usually for JSON files I'm having a huge printWidth..

Reply:
That's not really how updating JSON works. One of the big advantages of JSON over (e.g. YAML) is it is easy to make a program to read, patch, and write the file back to disk. `JSON.stringify` is the standard tool for this job and I don't think a couple of line breaks added by Prettier warrant the the added complexity of extra config and linters.

Reply:
For what it's worth, the only reason I disable prettier for package.json in my projects is because of this behavior it has with long string keys, since it only ever seems to introduce diffs